autoload/codi.vim

```diff
@@ -140,9 +140,14 @@ if has('unix')
       return ['script', '-qfec', tmp_bin, '/dev/null']
     endfunction
   endif
-else
-  call s:log ('Windows detected, erroring out')
-  call s:err('Codi does not support Windows yet.')
+else  
+  call s:log ('Windows detected')
+  if !s:nvim
+    call s:err('Only nvim is supported for Codi on windows')
+  endif
+  function! s:scriptify(bin)
+      call s:log('Scriptify is not set up for windows')
```

Comment:
Can we make this `s:err`, so it's obvious we've done something wrong if we call the function?

+ indentation nit:

```suggestion
    call s:err('Scriptify is not set up for windows')
```